app/views/bets/_display_bets.html.erb

```diff
@@ -5,10 +5,16 @@
         <tr>
           <th class="col-md-2">Created By</th>
           <th class="col-md-2">Created at</th>
+          <th class="col-md-2">Expires at</th>
           <th class="col-md-2">Total amount bet</th>
-          <th class="col-md-6">Bet description</th>
+          <th class="col-md-4">Bet description</th>
         </tr>
       </thead>
+      <tbody>
+        <% bets.each do |bet| %>
+          <%= render 'bet', bet: bet %>
```

Comment:
You can replace this with `<%= render bet %>` as Rails will infer that the `_bet` partial can be used to render a Bet model. I don't feel strongly about one way or the other, just thought you should know in case you encounter it